### ISSUE TITLE
Major issue: torchmetrics evaluation not working! 

### ISSUE THREAD

**Issue by baumgach** [baumgach/mimeta-cbim-classifier#1]
Don't run torchmetrics with the logits. Run them with the appropriate inputs!
